### DISCUSSION TITLE
Updating data on Game Menu

### DISCUSSION THREAD

**Discussion by ddiliberto123** [AlmasB/FXGL#1363] [Q&A] [answered]
How would I re-initialize/update the game menu? I've got a data singleton that stores a value that is accessed by the game menu and displayed to the user. However, I've created my own custom game menu and override this method inside the `initSettings` which leads the data to be accessed at the start of the program and never again. Are there any methods that I haven't found that could be used to re-access the information within my singleton upon opening the game menu?

**Comment by AlmasB** [comment, marked as answer]
Hi, if these are UI objects, such as JavaFX Text, you can bind its textProperty() to data properties in your singleton. Alternatively, feel free to post links to the source code if you seek a specific solution.
